wagtailio/core/migrations/0056_auto_20221114_2049.py

```diff
@@ -1,30 +1,1589 @@
 # Generated by Django 3.2.16 on 2022-11-14 20:49
```

Comment:
suggestion: rename this migration. we try to steer away from auto-named migrations